lms/static/styles/components/_SubmitGradeForm.scss

```diff
@@ -37,7 +37,7 @@
     font-weight: 500;
     border: 1px solid var.$grey-3;
     background-color: var.$grey-1;
-    transition: 0.2s;
+    transition: background-color 0.2s;
```

Comment:
this was causing a funny looking issue w/ outline so making it specific to the background color fixed that issue.
best to only apply a transition on only the things we want to transition.